nautobot/users/models.py

```diff
@@ -326,3 +328,12 @@ class Meta:
 
     def __str__(self):
         return f"{self.owner.username} - {self.view} - {self.name}"
+
+    def clean(self):
+        super().clean()
+
+        # If this SavedView is set to a global default, all other saved views related to this view name should not be the global default.
+        if self.is_global_default:
+            SavedView.objects.filter(view=self.view).exclude(pk=self.pk).update(is_global_default=False)
+            # If the view is set to Global default, is_shared should be true regardless
+            self.is_shared = True
```

Comment:
This feels like it belongs in `save()` rather than `clean()` to me since it has side effects.